jms/src/main/scala/akka/stream/alpakka/jms/JmsConsumerStage.scala

```diff
@@ -19,7 +19,7 @@ import scala.concurrent.Future
 import scala.util.control.NonFatal
 import scala.util.{Failure, Success}
 
-final class JmsSourceStage(settings: JmsConsumerSettings)
+final class JmsConsumerStage(settings: JmsConsumerSettings)
```

Comment:
Ah, could you make it `private[jms]`, please.

Reply:
For all the graph stage implementations I assume